### PR TITLE
feat: fastlane

### PR DESCRIPTION
**Pull request by anhle10051996**

### CODE REVIEW

**Review by loc-nguyenthien — APPROVED** [review]

File: template/scripts/build-app.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#!bin/bash
+
+# Author: Le Duc Tung
```
Comment (baonguyenhsts) [inline template/scripts/build-app.sh]: Change the author

File: template/scripts/code-push.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#!bin/bash
+
+# Author: Le Duc Tung
```
Comment (baonguyenhsts) [inline template/scripts/code-push.sh]: Change the author

**Review by baonguyenhsts — CHANGES_REQUESTED** [review]

File: template/scripts/code-push.sh [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+# Username: ledutu
```
Comment (ThinhKimVo) [inline template/scripts/code-push.sh]: remove

Comment (anhle10051996) [reply template/scripts/code-push.sh]: Updated

File: template/scripts/code-push.sh (hunk 2 above)
Comment (anhle10051996) [reply template/scripts/code-push.sh]: Updated

File: template/scripts/build-app.sh (hunk 1 above)
Comment (anhle10051996) [reply template/scripts/build-app.sh]: Updated

**Review by ThinhKimVo — APPROVED** [review]

**Review by huydosgtech — APPROVED** [review]

**Review by baonguyenhsts — APPROVED** [review]

**Review by ngochuyduong — APPROVED** [review]

File: template/scripts/build-app.sh [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+TYPE=$1
+
+function buildApp() {
+  cd ios && fastlane build type:build env:$1 && cd .. && cd android && fastlane build type:build env:$1
+}
+
+function main() {
+
+  if [ -z $TYPE ]; then
+    read -p 'Enter your environment: ' env
```
Comment (TranQuangPhi) [inline template/scripts/build-app.sh]: I suggest that we should list all environments to make it cleaner, like this: Enter your environment (dev, staging, production)

**Review by TranQuangPhi — APPROVED** [review]

File: template/scripts/build-app.sh (hunk 4 above)
Comment (anhle10051996) [reply template/scripts/build-app.sh]: You mean cleaner or clearer ?

Comment (TranQuangPhi) [reply template/scripts/build-app.sh]: I mean clearer. I'm sorry if I confused you about that

Comment (anhle10051996) [reply template/scripts/build-app.sh]: Updated

**Pull request merged**